skins/cat17/address-form/src/store.ts

```diff
@@ -93,12 +92,12 @@ const store: StoreOptions<AddressState> = {
 			state.isValidating = true;
 		},
 		[FINISH_ADDRESS_VALIDATION](state, payload) {
-            if (payload.status === 'ERR') {
+            if (payload.status === ValidationStates.ERR) {
                 ADDRESS_FIELD_VALIDATOR_NAMES.forEach(name => {
                     if (state.form[name].dataEntered === true) {
                         state.form[name] = {
                             ...state.form[name],
-                            isValid: state.form[name].isValid !== false && !payload.messages[name] ? Validity.VALID : Validity.INVALID
+                            isValid: state.form[name].isValid = !payload.messages[name] ? Validity.VALID : Validity.INVALID
```

Comment:
this doesn't seem right. Did you mean:
`isValid:  !payload.messages[name] ?  Validity.VALID : Validity.INVALID`
which will be better off as:
`isValid:  payload.messages[name] ?  Validity.INVALID : Validity.VALID`
or is `=` supposed to be `===` ?